metricbeat/tests/system/test_system.py

```diff
@@ -3,11 +3,14 @@
 import unittest
 import metricbeat
 
-SYSTEM_CPU_FIELDS = ["idle", "iowait", "irq", "load", "nice", "softirq",
-                     "steal", "system", "system_p", "user", "user_p"]
+SYSTEM_CPU_FIELDS = ["idle_p", "iowait_p", "irq_p", "load", "nice_p",
+                     "softirq_p", "steal_p", "system_p", "user_p"]
 
-SYSTEM_CORE = ["core", "idle", "iowait", "irq", "nice", "softirq",
-                "steal", "system", "system_p", "user", "user_p"]
+SYSTEM_CPU_ALL_FIELDS = ["idle_p", "idle", "iowait_p", "iowait", "irq_p", "irq", "load", "nice_p", "nice",
```

Comment:
we can to this also at a later stage, but I was thinking if it would make sense to group the ticks together. `system.cpu.ticks.idle` and `system.cpu.ticks.steal`.